chain/vm/validation_test.go

```diff
@@ -25,3 +25,8 @@ func TestValueTransfer(t *testing.T) {
 	suites.AccountValueTransferFromUnknownToKnownAccount(t, factory, 0)
 	suites.AccountValueTransferFromUnknownToUnknownAccount(t, factory, 0)
 }
+
+func TestPaymentChannelActor(t *testing.T) {
+	factory := validation.NewFactories()
+	suites.PaymentChannelCreateSuccess(t, factory, 1120)
```

Comment:
Is that last parameter the gas value? Explicit here because implementations differ? I think we can find a better pattern than that, but it needn't block this PR.

Reply:
Yup that's gas :fuelpump: . Agreed, this pattern is far from ideal but exists since gas usage isn't consistent across implementations.